subprojects/docs/src/main/resources/header.html

```diff
@@ -102,7 +102,8 @@ <h3 id="using-gradle">Using Gradle</h3>
             <li><a href="/userguide/installation.html">Installing Gradle</a></li>
             <li><a class="nav-dropdown" data-toggle="collapse" href="#upgrading-gradle" aria-expanded="false" aria-controls="upgrading-gradle">Upgrading Gradle</a>
                 <ul id="upgrading-gradle">
-                    <li><a href="/userguide/upgrading_version_4.html">version 4.x to 5.0</a></li>
+                    <li><a href="/userguide/upgrading_version_5.html">version 5.0 to 5.X</a></li>
```

Comment:
Maybe just "version 5.X" so we don't have to update it every release? Note that we don't have the luxury of asciidoctor variable replacement here. there are other ways to replace versions but they get all JavaScripty and I'm just not sure the complexity is worth it. WDYT?